### ISSUE TITLE
cant merge modelengine and itemsadder texturepack

### ISSUE THREAD

**Issue by Gangleaderyt** [PluginBugs/Issues-ItemsAdder#3361]
### Terms

- [x] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

gangleader_

### What happened?

So basically items adder wont merge modelengine texturepack there is no errors or anything and i am sure i did the config correctly

### Steps to reproduce the issue

1.start the server
2 join the game
3 model engine models arent being loaded

### Server version

git-paper-196 MC 1.20.1

### ItemsAdder Version

ItemsAdder 3.6.3-beta-8 cant use newer server crashes

### ProtocolLib Version

ProtocolLib version 5.1.0

### LoneLibs Version

1.0.45 LoneLibs version

### FULL server log

[2024-01-22-5.log](https://github.com/PluginBugs/Issues-ItemsAdder/files/14014037/2024-01-22-5.log)

**Comment by LoneDev6** [comment]
Did you follow the tutorial? The merging is automatic once you run /iazip.
https://itemsadder.devs.beer/compatibility-with-other-plugins/compatible/modelengine#how-to-add-compatibility

If you still have issues please explain in detail what you mean by "model engine models arent being loaded".
Post some screenshots, also try to disable ItemsAdder, apply the modelengine pack manually and make sure the models actually work.

**Comment by Gangleaderyt** [comment]
Yes the models work
But when i spawn the bosses that use those models all i see is horse armor
And i followed the tutorial

**Comment by Gangleaderyt** [comment]
When i use only modelengine pack without itemsadder everything works normally

**Comment by LoneDev6** [comment]
Post the output of `/iainfo` command

**Comment by Gangleaderyt** [comment]
[13:35:05 INFO]:
Path: /container/plugins/Items
Adder/output/generated.zip
[13:35:05 INFO]:
Download: https://www.dropbox.
com/scl/fi/idfwg0s25zpi12oqm3zhm/cubemaniatest.zip?r
1key=tajulneuwd1exfdbgyuyirdbl&raw=1#170595786114795
7d
[13:35:05 INFO]:
[13:35:05 INFO]:
Size: 1 MB/250 MB
Last update: 2024-01-22 21:15:
40
[13:35:05 INFO]:
Host type: external
[13:35:05 INFO]:
Data
[13:35:05 INFO]:
[13:35:05 INFO]:
[13:35:05
INFO]:
[13:35:05 INFO]:
[13:35:05
INFO]:
[13:35:05
INFO]:
[13:35:05
INFO]:
Items: 226
Items in lia: 48
Namespaces: 8
Materials: 8
REAL block IDs: 0/191
REAL_NOTE block IDs: 0/750
REAL_TRANSPARENT block IDs: 0/
63
[13:35:05 INFO]:
REAL_WIRE block IDs: 0/127
[13:35:05
INFO]:
FIRE block IDs: 0/14
[13:35:06
INFO]:
Font images: 198/6608
[13:35:06
INFO]:
Loaded chunks (1587)
[13:35:06
INFO]:
Armorstands: 0
[13:35:06 INFO]:
Mobs: 0

**Comment by Gangleaderyt** [comment]
Some stuff might br missing bc im on phone rn

**Comment by Gangleaderyt** [comment]
I also changed to dropbox so it kinda works the problem is when i add a model it stopsworking

**Comment by LoneDev6** [comment]
You probably didn't regenerate your pack using `/iazip` command or forgot to reupload the pack in case you're using Dropbox.
In this case I suggest you to use [selfhost](https://itemsadder.devs.beer/plugin-usage/resourcepack-hosting/resourcepack-self-hosting) or to read about [lobfile](https://itemsadder.devs.beer/plugin-usage/resourcepack-hosting/lobfile-hosting) hosting.

Can you try using LobFile? It's easy to setup, let me know

**Comment by Gangleaderyt** [comment]
I used /iazip it still doesnt merge with modelengine

**Comment by Gangleaderyt** [comment]
I also uploaded to dropbox

**Comment by LoneDev6** [comment]
> I also uploaded to dropbox

After you uploaded to dropbox did you copy the new URL and paste it into the config.yml, then run /iareload?
As shown in [the tutorial](https://itemsadder.devs.beer/plugin-usage/resourcepack-hosting/resourcepack-on-dropbox)?

**Comment by Gangleaderyt** [comment]
Yes

**Comment by LoneDev6** [comment]
When i try to reach the pack I get a permission error.
Might be the issue.
Make sure that your pack files are public when you upload them.

![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/27242001/0ef46a23-21e7-4d2e-8d7d-220967459c3e)

**Comment by Gangleaderyt** [comment]
Oh alright

**Comment by Gangleaderyt** [comment]
I will try when i get home

**Comment by Gangleaderyt** [comment]
i see what u meant but when i use the pack it loads just no modelengine textures

**Comment by LoneDev6** [comment]
This means that the modelengine files are not bundled into the pack for some reason.
Can you download the pack from the /iainfo pack url and check if the modelengine folder is in the assets folder?
